pkg/base1/cockpit.js

```diff
@@ -1606,9 +1630,7 @@ function basic_scope(cockpit, jquery) {
                 callback.call(self, row, x, n);
             }
 
-            var event = document.createEvent("CustomEvent");
-            event.initCustomEvent("notify", false, false, [ x, n ]);
-            self.dispatchEvent(event, x, n);
+            self.dispatchEvent("notify", x, n);
```

Comment:
I'm not sure, but this seems like it changes the behavior:
https://github.com/cockpit-project/cockpit/pull/4145/files#diff-34cbdb779d20a4a75591fcba50bd25d4R183